### DISCUSSION TITLE
Issue with shortcut

### DISCUSSION THREAD

**Discussion by mikeerickson** [ttscoff/bunch-docs#229] [Q&A]
I have assigned a shortcut to one of my scripts to see how it works.

The shortcut appears correctly in the Bunch menu, and when I trigger the script shortcut, the bunch menu bar icon with toggle between gray and green, so it seems to be doing something.

However, the script does not do anything, but if I select the script from the menu to trigger the script, everything works as expected.

I am using Bunch Beta
Version 1.4.9 (147)

**Comment by ttscoff** [comment]
Are you triggering the bunch menu before pressing the shortcut for the individual bunch? It's a 2-step process requiring a global bunch shortcut AND an individual bunch shortcut.

-Brett
________________________________
From: Mike Erickson ***@***.***>
Sent: Saturday, July 16, 2022 4:51:37 PM
To: ttscoff/bunch ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ttscoff/bunch] Issue with shortcut (Discussion #229)


I have assigned a shortcut to one of my scripts to see how it works.

The shortcut appears correctly in the Bunch menu, and when I trigger the script shortcut, the bunch menu bar icon with toggle between gray and green, so it seems to be doing something.

However, the script does not do anything, but if I select the script from the menu to trigger the script, everything works as expected.

I am using Bunch Beta
Version 1.4.9 (147)

**Reply by mikeerickson** [reply]
Hey Brett, yes I am triggering bunch and then the script keystroke.  I will give it another whirl tomorrow morning to confirm.
